skribenten-backend/src/main/kotlin/no/nav/pensjon/brev/skribenten/routes/Brev.kt

```diff
@@ -23,49 +22,31 @@ fun Route.bestillBrevRoute(
     tjenestebussIntegrasjonService: TjenestebussIntegrasjonService,
     brevmetadataService: BrevmetadataService,
     safService: SafService,
+    penService: PenService,
```

Comment:
Kanskje lage en samle-service for bestilling av legacy-brev? Type BestillBrevLegacyService? 
På den måten så får du ryddet ut en god del kode fra route-filen, og du slipper å sende med alle disse servicene hele veien i alle kall.

Reply:
Done. Det ble nice.